resources/assets/templates/vote/voteDetailComment.hbs

```diff
@@ -5,6 +5,7 @@
     <div class="vote-comment-body">
         <h4>{{ meta.user.first_name }} {{ meta.user.last_name }} <span>{{ createdDate }}</span></h4>
         <p class="vote-comment-text">{{ model.content_origin }}</p>
+        <a id="comment_reply">reply with quote</a>
```

Comment:
There are can be more then one comment block on page. So every of comment block will have link with id="comment_reply" that is wrong. Id's should be unique. So use class name instead (don't forget to replace in frontend app)